Assignment/starter_code/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,69 @@
 // 1.	Slide the header up when it is clicked on.
-
+$('h1').on('click', function(){
+	$('h1').slideUp();
+});
 // 2.	Remove the purplebox class
+$('#box14').removeClass('purplebox');
+
+// Do you need to target the specfic box, why does 'body' not work here?
 
 // 3.	Hide all of the div’s in #row4
 
+$('#row4 div').hide();
+
 // 4.	Add a paragraph that says “I have been added” to all boxes with class “add-para” (Hint: you'll need to look up the append() method)
 
+$('.add-para').append('I have been added');
+
 // 5.	Add a class in your css and add it to all items with the class .box
 
+$('.box').addClass('fun')
```

Comment:
As you can see it didn't cause an error, but remember to try and close out each statement with a `;` :)